localization/localization_util/test/test_smart_pose_buffer.cpp

```diff
@@ -44,10 +44,6 @@ bool compare_pose(
 
 class TestSmartPoseBuffer : public ::testing::Test
 {
-protected:
-  void SetUp() override {}
-
-  void TearDown() override {}
 };
 
 TEST_F(TestSmartPoseBuffer, interpolate_pose)  // NOLINT
```

Comment:
I think we can remove the unused class.

```
class TestSmartPoseBuffer : public ::testing::Test
{
};

TEST_F(TestSmartPoseBuffer, interpolate_pose)  // NOLINT
```

```
TEST(TestSmartPoseBuffer, interpolate_pose)  // NOLINT
```

Reply:
Thank you for you suggestion.
I fixed at e54a112 .